cms/djangoapps/contentstore/features/component.py

```diff
@@ -74,8 +74,7 @@ def see_a_problem_component(step, category):
                 'No problem was added to the unit.')
 
     problem_css = 'li.studio-xblock-wrapper div.xblock-student_view'
-    actual_text = world.css_text(problem_css)
-    assert_in(category.upper(), actual_text)
+    assert_true(world.css_contains_text(problem_css, category.upper()))  # pylint: disable=no-value-for-parameter
```

Comment:
That is odd. Wouldn't it be safer to just call .lower() on both strings?

On Thu, Jun 11, 2015 at 5:05 PM, Christine Lytwynec <
notifications@github.com> wrote:

> In cms/djangoapps/contentstore/features/component.py
> <https://github.com/edx/edx-platform/pull/8471#discussion_r32268140>:
>
> > @@ -74,8 +74,7 @@ def see_a_problem_component(step, category):
> >                  'No problem was added to the unit.')
> >
> >      problem_css = 'li.studio-xblock-wrapper div.xblock-student_view'
> > -    actual_text = world.css_text(problem_css)
> > -    assert_in(category.upper(), actual_text)
> > +    assert_true(world.css_contains_text(problem_css, category.upper()))  # pylint: disable=no-value-for-parameter
>
> Another question... I know it was there before, but do you know why
> .upper() is used here? Is it making an assumption that the text in the
> problem is always uppercase?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/edx-platform/pull/8471/files#r32268140>.
>